packages/client/src/App.js

```diff
@@ -12,6 +15,9 @@ function App() {
       <Router>
         <Routes>
           <Route path="/" element={<LandingPage />} />
```

Comment:
should be `<HomePage />` here

Reply:
it was already there @shpomp should I change it to HomePage ?